tools/clang/lib/SPIRV/SpirvBasicBlock.cpp

```diff
@@ -15,7 +15,7 @@ namespace spirv {
 
 SpirvBasicBlock::SpirvBasicBlock(llvm::StringRef name)
     : labelId(0), labelName(name), mergeTarget(nullptr),
-      continueTarget(nullptr) {}
+      continueTarget(nullptr), debugScope(nullptr) {}
 
 SpirvBasicBlock::~SpirvBasicBlock() {
```

Comment:
You also need to release memory for the debugScope you've added to the basic block.

Reply:
done